swagger.yaml

```diff
@@ -390,6 +402,8 @@ components:
           type: string
         tocomanId:
           type: integer
+        status:
```

Comment:
Should this also be a required property? Project without status is quite confusing idea